src/app.test.ts

```diff
@@ -465,15 +523,14 @@ test('GET /styles/:styleId when style exists returns altered style', async (t) =
   t.equal(responseGet.statusCode, 200)
 
   // This will change if a style fixture other than good-stylejson/good-simple.json is used
-  const expectedTilesetId = 'yqtx3fxnp2vdyssc82ew4f377g4y0njk' // generated from getTilesetId in lib/utils.ts
+  const expectedTilesetId = 'kgbnbb3mck9jekk1ct6f3anjpe804rnv' // generated from getTilesetId in lib/utils.ts
```

Comment:
Is there a way to avoid hard-coding this and checking against this? What do we want to test here? Hard-coding values like this in a test can be frustrating when changing a test later on.

Reply:
> What do we want to test here?

The goal of this test to make sure that the returned style has altered its `sources` properly i.e. remapping the `url` property to point to the resource url in the map server. The url ends with the `tilesetId`, hence why we want to expect a specific id.

> Is there a way to avoid hard-coding this and checking against this?

I suppose the actual id is not really that important for this test, as I'm writing out this response 😅  could simplify it by just checking to make sure the `url` is mapped to the origin of the map server as opposed to its original url, if that makes sense

Reply:
Ok so I think what we actually want to test is that both the style is created and that tilesets are created for the sources in the style. I think this test should follow the source.url and check the tileset exists - that is a more robust test of behaviour.

Reply:
Addressed via https://github.com/digidem/mapeo-mapserver/pull/19/commits/24b59db17d289b423c19fafb08eab7e4df6f1df1